pages/transactions/index.tsx

```diff
@@ -100,7 +202,160 @@ const TransactionsPage: React.FunctionComponent = () => {
            </Box>
        );
    }
+
+   const addPopoverContent = () => {
```

Comment:
With your current implementation, we can't see live updates of the transactions if new ones are added, and I think this is a feature Cindy and I wanted to see. It's expensive to add the data to the firebase backend, pull the data again, then refresh the table. So, a potential method I thought of is updating the list of transactions/useState variables you're using to display the data on the tables and also updating the backend. If you have a better idea or implementation, let me know!